src/System.Drawing.Common/src/System/Drawing/Design/CategoryNameCollection.cs

```diff
@@ -3,68 +3,23 @@
 // See the LICENSE file in the project root for more information.
 
 using System.Collections;
+using System.Security.Permissions;
 
 namespace System.Drawing.Design
 {
-    /// <summary>
```

Comment:
I reverted the change to the doc comments.
> We don't actually maintain any other doc comments

What about https://docs.microsoft.com/en-us/dotnet/api/system.drawing.fontfamily?view=netframework-4.7

These docs seem to be a lot better and more comprehensive that the docs comments in the source